scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -33,7 +33,7 @@
                             variant_id=variant._id) }}">
   {{ variant.sub_category|upper }}({{ variant.chromosome }}{{ variant.cytoband_start }}-{{ variant.chromosome }}{{ variant.cytoband_end }})
   {% endif %}
-  </a>  
+  </a>
```

Comment:
🏴

Reply:
Better without the space. I'm working on testing the entire sidebar now. Almost ready (if it works)